lib/linguist/languages.yml

```
@@ -5512,6 +5512,15 @@ RAML:
   extensions:
   - ".raml"
   language_id: 308
+RBS:
+  type: programming
+  ace_mode: text
```

Both [CodeMirror](https://codemirror.net/5/mode/ruby/index.html) and [ACE](https://ace.c9.io/build/kitchen-sink.html)'s Ruby modes do a decent job of highlighting RBS syntax, so they're worth reusing:

```suggestion
  ace_mode: ruby
  codemirror_mode: ruby
  codemirror_mime_type: text/x-ruby
```